src/main/java/com/hedera/exchange/ExchangeRateTool.java

```diff
@@ -165,7 +164,7 @@ private static void execute() throws Exception {
     private static void updateTransactionFileForNetwork(final String networkName,
             final AccountId operatorId,
             final ExchangeRate exchangeRate,
-            final Rate midnightRate,
+            final ExchangeRate midnightRate,
```

Comment:
should be renamed to `midnightExchangeRate`

Reply:
agreed